docs/en/engines/table-engines/integrations/embedded-rocksdb.md

```diff
@@ -15,11 +15,13 @@ CREATE TABLE [IF NOT EXISTS] [db.]table_name [ON CLUSTER cluster]
     name1 [type1] [DEFAULT|MATERIALIZED|ALIAS expr1],
     name2 [type2] [DEFAULT|MATERIALIZED|ALIAS expr2],
     ...
-) ENGINE = EmbeddedRocksDB PRIMARY KEY(primary_key_name)
+) ENGINE = EmbeddedRocksDB([ttl, read_only]) PRIMARY KEY(primary_key_name)
 ```
 
-Required parameters:
+Engine parameters:
 
+- `ttl` - time to live. TTL is accepted in seconds. Not specifying/passing or non-positive TTL behaves like TTL = infinity.
+- `read_only` - when `read_only` be set `true` opens in the usual read-only mode. Compactions will not be triggered(neither manual nor automatic), so no expired entries removed.
```

Comment:
This is not a TTL only feature, we can use it for non-ttl database also.
Let's change the description to

```suggestion
- `read_only` - when `read_only` is set to `true`, read-only mode is used. For storage with TTL, compaction will not be triggered (neither manual nor automatic), so no expired entries are removed.
```